src/test/scala/wdltool/graph/WomDotGraphTest.scala

```diff
@@ -0,0 +1,41 @@
+package wdltool.graph
+
+import java.util.concurrent.atomic.AtomicInteger
+import org.scalatest.{FlatSpec, Matchers}
+import wdl4s.wom.graph.Graph
+
+trait WomDotGraphTest extends FlatSpec with Matchers {
+
+  def cases: List[WomDotGraphTestCase]
+
+  behavior of "womgraph"
+
+  def tests() = {
+    cases foreach { testCase =>
+      it should s"draw the right DOT graph for ${testCase.name}" in {
+        val womGraph = new WomGraph(testCase.name, testCase.graph)
+
+        val standardResult = standardizifyResult(womGraph.digraphDot)
+        val standardExpectation = standardizifyResult(testCase.dotExpectation)
+
+        standardResult should be(standardExpectation)
+      }
+    }
+  }
+  
+  def standardizifyResult(dot: String) = {
+    val regex = "(NODE|PORT)-?[0-9]*".r
+    val matches = regex.findAllIn(dot).toList.distinct.map(_.toString)
+    val currentMatchId = new AtomicInteger(0)
```

Comment:
it doesn't need to be, but it has an `getAndIncrement` method that's nicer than a two line get and set on a regular integer